akka-http-core/src/main/scala/akka/http/impl/engine/HttpConnectionTimeoutException.scala

```diff
@@ -0,0 +1,5 @@
+package akka.http.impl.engine
+
+import scala.util.control.NoStackTrace
+
+class HttpConnectionTimeoutException(msg: String) extends RuntimeException(msg) with NoStackTrace
```

Comment:
Since it exists since 2014 (45ddb7b), does it need a 2009-2014 header? (nitpicking :))